pipelines/jobs/configurations/jdk11u.groovy

```diff
@@ -1,62 +1,20 @@
 targetConfigurations = [
-        "x64Mac"      : [
-                "hotspot",
-                "openj9"
-        ],
-        "x64MacXL"    : [
-                "openj9"
-        ],
-        "x64Linux"    : [
-                "hotspot",
-                "openj9",
-                "corretto",
-                "dragonwell"
-        ],
-        "x64Windows"  : [
-                "hotspot",
-                "openj9",
-                "dragonwell"
-        ],
-        "x64WindowsXL"  : [
-                "openj9"
-        ],
-        "x32Windows"  : [
-                "hotspot"
-        ],
-        "ppc64Aix"    : [
-                "hotspot",
-                "openj9"
-        ],
-        "ppc64leLinux": [
-                "hotspot",
-                "openj9"
-        ],
-        "s390xLinux"  : [
-                "hotspot",
-                "openj9"
-        ],
-        "aarch64Linux": [
-                "hotspot",
-                "openj9"
-        ],
-        "arm32Linux"  : [
-                "hotspot"
-        ],
-        "x64LinuxXL"     : [
-                "openj9"
-        ],
-        "s390xLinuxXL"     : [
-                "openj9"
-        ],
-        "ppc64leLinuxXL"     : [
-                "openj9"
-        ],
-        "aarch64LinuxXL": [
-                "openj9"
-        ],
-        "riscv64Linux": [
-                "openj9"
-        ]
+        "x64Mac"      : [	"hotspot",	"openj9"					],
+        "x64MacXL"    : [			"openj9"					],
+        "x64Linux"    : [	"hotspot",	"openj9",	"dragonwell",	"corretto"	],
+        "x64Windows"  : [	"hotspot",	"openj9",	"dragonwell"			],
+        "x64WindowsXL": [			"openj9"					],
+        "x32Windows"  : [	"hotspot"							],
+        "ppc64Aix"    : [	"hotspot",	"openj9"					],
+        "ppc64leLinux": [	"hotspot",	"openj9"					],
+        "s390xLinux"  : [	"hotspot",	"openj9"					],
+        "aarch64Linux": [	"hotspot",	"openj9"					],
+        "arm32Linux"  : [	"hotspot"							],
+        "x64LinuxXL"  : [			"openj9"					],
+        "s390xLinuxXL": [			"openj9"					],
+        "ppc64leLinuxXL":[			"openj9"					],
+        "aarch64LinuxXL":[			"openj9"					],
```

Comment:
`:[` not lined up with the others

Reply:
Honestly, I wasn't too bothered about it purely because the XL variants will be disappearing soon but I'll fix :-)